### ISSUE TITLE
Accessibility checkup

### ISSUE THREAD

**Issue by loogart** [loogart/impact-canada#53]
- [ ] Broken Skip link (lang toggle) http://wave.webaim.org/report#/https://impact.canada.ca/
- [ ] Main tag (Broken ARIA reference)

**Comment by loogart** [comment]
@djsutter hey Duncan, can we fix the accessibility issues based off this report? there are multiple broken skip links and broken ARIA references. There seems to be a double for **skip to main content** and **skip to about this site** and the **skip to language toggle** is broken.

Here are sample reports from the WAVE tool online. This applies to all main pages and challenge pages.
 
**Main pages**
http://wave.webaim.org/report#/https://impact.canada.ca/en
http://wave.webaim.org/report#/https://impact.canada.ca/en/about
http://wave.webaim.org/report#/https://impact.canada.ca/en/how-it-works
https://impact.canada.ca/en/news
https://impact.canada.ca/en/partner-with-us

**Challenge page example** 
http://wave.webaim.org/report#/https://impact.canada.ca/en/challenges/crush-it

**Comment by djsutter** [comment]
I fixed the skip links on the impact.qualivera site but you might want to have a look to see if it's what you're expecting. I found that the WxT theme includes a built-in method to set the top two links from the GUI, and the 3 links which follow were provided by the template files. I have deleted those. So what remains is the first two links to main content and language toggle. If it looks good, I can apply part of the fix to the live site right away. Then we can look at the other issues being reported.
